dungeon/src/contrib/level/generator/graphBased/levelGraph/LevelNode.java

```diff
@@ -72,8 +78,8 @@ public boolean connect(final LevelNode other) {
     List<Direction> freeDirections = possibleConnectDirections(other);
     if (!freeDirections.isEmpty()) {
       Collections.shuffle(freeDirections);
-      if (other.connect(this, Direction.opposite(freeDirections.get(0))))
-        return connect(other, freeDirections.get(0));
+      if (other.connect(this, Direction.opposite(freeDirections.getFirst())))
+        return connect(other, freeDirections.getFirst());
```

Comment:
@tgrothe Ich hasse solche on-the-fly Aktionen! Bitte mach einen eigenen PR dafür auf, aber nicht im Vorbeigehen irgendwo irgendwas "verbessern".

Ja, ich finde `getFirst()` sinnvoller als `get(0)`, aber das gehört hier nicht rein!

Reply:
Ok. :) Ich hab ja jetzt auch schon Übung darin, die Kette in Gang zu setzen. ;)

Das nehme ich hier wieder raus, und erstelle ein Issue dafür.

Reply:
Kein neues Issue! Einfach einen PR. Das reicht völlig - die von @AMatutat mal eingeführte Policy, dass es immer erst ein Ticket geben muss, möchte ich so nicht weiterführen. 

- [ ] Wenn es ein Problem gibt, was man sofort fixen kann -> Pull Request.
- [ ] Wenn es nicht sofort gelöst werden kann -> Ticket (als Merkhilfe).

Reply:
Bin ich auch für. 👍🏼 #1463 ist auf. Schönes WoEn erst mal.